js/igv-create.js

```diff
@@ -328,7 +332,13 @@ var igv = (function (igv) {
             browser.$searchInput = $('<input type="text" placeholder="Locus Search">');
 
             browser.$searchInput.change(function (e) {
-                browser.parseSearchInput( $(e.target).val().toLowerCase() );
+              var result = $(e.target).val();
```

Comment:
Yeah, I think It is necessary since I can not change $(e.target).val() in the code lines below. So I used result instead.